engine/classes/Elgg/Database/Annotations.php

```diff
@@ -369,7 +369,7 @@ function getEntities(array $options = array()) {
 			$options['annotation_created_time_lower']);
 	
 		if ($time_wheres) {
-			$options['wheres'] = array_merge($options['wheres'], $time_wheres);
+			$options['wheres'][] = $time_wheres;
```

Comment:
I did a testing like this:
```
$options['wheres'] = array("1","2"); // existing array
$options['wheres'][] = "3";
var_dump($options['wheres']);
```
Output
```
array(3) { [0]=> string(1) "1" [1]=> string(1) "2" [2]=> string(1) "3" }
```
I saw that $options['wheres'] was having all the 3 values. So any "String" when added to the existing array using [] process gets amended as an value.

if I do something like this:
```
$options['wheres'] = array("1","2");
$options['wheres'][] = array("3");
var_dump($options['wheres']);
```
Output
```
array(3) { [0]=> string(1) "1" [1]=> string(1) "2" [2]=> array(1) { [0]=> string(1) "3" }}
```
then here I saw that 1,2 are array value and then 3rd value was a new array within the array.

So @jdalsem what you are saying about adding ```wheres``` to the defaults as an array will not be correct as it will get stored as an array within the array.